### ISSUE TITLE
1.20.4 event handling changes in Draggable.js cause "undefined" events to be thrown

### ISSUE THREAD

**Issue by flipace** [greensock/GreenSock-JS#250]
We're using gsap inside a cordova application, and after updating to 1.20.4 we experienced an error which was thrown inside cordova.js where they do something like this:

`document.addEventListener = (evt, ...) => { const t = evt.toLowerCase(); }`

Since gsap 1.20.4 it seems like events which are "undefined" are triggered, which leads to an error since undefined can't be "lower cased".

I checked what changed in your latest version and it seems like these lines here cause the problem:
https://github.com/greensock/GreenSock-JS/commit/2802e96d94358b2ad28381ca83ac71de7f90f95e#diff-d3dfc8968ad27119dcc88d29f10cde54L803

**Comment by jackdoyle** [comment]
I think I see the problem and I can just add " || type" after the event lookup. In other words:

```javascript
//BEFORE:
element.addEventListener(_touchEventLookup[type], func, capture);

//AFTER:
element.addEventListener(_touchEventLookup[type] || type, func, capture);
```
Here's an uncompressed version with that edit in place: 
https://s3-us-west-2.amazonaws.com/s.cdpn.io/16327/Draggable-latest-beta.js

Does that resolve things for you?

**Comment by jackdoyle** [comment]
This has been resolved in 1.20.5 (just pushed live). Enjoy!
